source/fundamentals/connection/mongoclientsettings.txt

```diff
@@ -476,6 +463,60 @@ This example specifies the following driver behavior in a MongoDB server:
 
 .. _mcs-ssl-settings:
 
+Socket Settings
+---------------
```

Comment:
mismatched anchor

Reply:
As per above - I think this is already fixed.